source/PluginDev/Assets/GooglePlayGames/Platforms/Native/NativeClient.cs

```diff
@@ -285,6 +285,71 @@ private static void LaunchBridgeIntent(IntPtr bridgedIntent)
                 AndroidJNIHelper.DeleteJNIArgArray(objectArray, jArgs);
             }
         }
+        
+        private AndroidJavaObject GetApiClient(GameServices services) {
```

Comment:
Can you replace this with
`using C = GooglePlayGames.Native.Cwrapper.InternalHooks;`

`return JavaUtils.JavaObjectFromPointer(C.InternalHooks_GetApiClient(services.AsHandle()));`